src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/ServiceController.php

```diff
@@ -128,7 +128,8 @@ public function overviewAction()
         $productionEntityEnabled = [];
         $privacyOK = [];
         foreach ($services as $service) {
-            $productionEntityEnabled[] = $service->isProductionEntitiesEnabled();
+            $productionEntityEnabled = $service->isProductionEntitiesEnabled();
+            $productionEntityEnabled[] = $productionEntityEnabled || $service->hasTestEntities();
```

Comment:
It needs to be an array because both a user can have multiple services.  Each service has it's own "is production entities enabled" trigger.  In the twig view we loop over the services to present them, and then take the corresponding boolean from the array to know whether or not to portray the "add a production entity"-button.

The question is moot though, by moving the logic to the service view object the entire array is redundant and removed 💯